backend/spec/finders/spree/admin/reports/orders/top_products_by_unit_sold_spec.rb

```diff
@@ -82,4 +84,22 @@
       end
     end
   end
+
+  context 'when top param is present' do
+    let(:top) { 1 }
+
+    it 'returns results for given top param' do
```

Comment:
I think the test doesn't cover all cases. Let's add orders with another variant (not `product1.master`).